tests/e2e/framework/kubernetes.go

```diff
@@ -37,13 +37,13 @@ const (
 )
 
 var (
-	namespace    = flag.String("namespace", "", "Namespace to use for testing (empty to create/delete temporary one)")
-	mixerHub     = flag.String("mixer_hub", os.Getenv(mixerHubEnvVar), "Mixer hub")
-	mixerTag     = flag.String("mixer_tag", os.Getenv(mixerTagEnvVar), "Mixer tag")
-	managerHub   = flag.String("manager_hub", os.Getenv(managerHubEnvVar), "Manager hub")
-	managerTag   = flag.String("manager_tag", os.Getenv(managerTagEnvVar), "Manager tag")
-	caHub        = flag.String("ca_hub", "", "Ca hub")
-	caTag        = flag.String("ca_tag", "", "Ca tag")
+	namespace  = flag.String("namespace", "", "Namespace to use for testing (empty to create/delete temporary one)")
+	mixerHub   = flag.String("mixer_hub", os.Getenv(mixerHubEnvVar), "Mixer hub")
+	mixerTag   = flag.String("mixer_tag", os.Getenv(mixerTagEnvVar), "Mixer tag")
+	managerHub = flag.String("manager_hub", os.Getenv(managerHubEnvVar), "Manager hub")
+	managerTag = flag.String("manager_tag", os.Getenv(managerTagEnvVar), "Manager tag")
+	//caHub        = flag.String("ca_hub", "", "Ca hub")
+	//caTag        = flag.String("ca_tag", "", "Ca tag")
```

Comment:
This is just to keep go compiler happy (unused variables). The program is not using the ca image currently, it's just penciled in for future use.